webapp/sass/layout/_post.scss

```diff
@@ -377,6 +377,7 @@ body.ios {
         }
 
         .custom-textarea {
+            bottom: 0;
```

Comment:
K just wanted to make sure. Unless I changed it, the post textbox, the post preview, and the edit post modal textbox are all .custom-textareas. I must've removed the `bottom: 0` because I thought it didn't do anything then